include/ibamr/ibamr_enums.h

```diff
@@ -213,6 +213,38 @@ enum_to_string<LevelSetOrder>(LevelSetOrder val)
     return "UNKNOWN_LEVEL_SET_ORDER";
 } // enum_to_string
 
+/*!
+ * \brief Enumerated type for different level set time stepping schemes
+ */
+enum LevelSetTimeStepping
+{
+    GAUSS_SEIDEL_PSEUDO_TS = 1,
+    RK1_TS = 2,
+    RK2_TS = 3,
+    UNKNOWN_LEVEL_SET_TS = -1
+}; // LevelSetTimeStepping
+
+template <>
+inline LevelSetTimeStepping
+string_to_enum<LevelSetTimeStepping>(const std::string& val)
+{
+    if (strcasecmp(val.c_str(), "GAUSS_SEIDEL") == 0) return GAUSS_SEIDEL_PSEUDO_TS;
+    if (strcasecmp(val.c_str(), "RK1") == 0) return RK1_TS;
+    if (strcasecmp(val.c_str(), "FORWARD_EULER") == 0) return RK1_TS;
+    if (strcasecmp(val.c_str(), "RK2") == 0) return RK2_TS;
```

Comment:
There is more than one RK2 --- could this be `EXPLICIT_MIDPOINT` or `EXPLICIT_TRAPEZOID` (or something else) as appropriate?

Reply:
I am using TVD RK2. I that `EXPLICIT_TRAPEZOID`?

Reply:
It is not `EXPLICIT_MIDPOINT` AFAICT from the top of my head.